### PR TITLE
tee: add `tee /dev/tty` mid-pipeline example

### PR DESCRIPTION
**Pull request by ticky**
- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/tee.md [hunk 1]
```diff
@@ -10,6 +10,10 @@
 
 `echo "example" | tee -a {{FILE}}`
 
+- Print standard input to the terminal, and also pipe it into another program for further processing:
+
+`echo "example" | tee /dev/tty | xargs printf "\033[1;34m%s\033[m\n"`
```
Comment (agnivade) [inline pages/common/tee.md]: /dev/tty in tokens `{{}}` please. Same for printf. Also, no need to colorize the output and make the example complicated. We want the pages to be simple and readable. printf "[%s]" is fine.

Comment (ticky) [reply pages/common/tee.md]: Updated in c6bb56d651b0e78b966f1daf18c10010334f473f. I wrapped the entire last command in the tokens; I hope that’s correct, but I couldn’t quite figure out from the docs what the intended approach for tokens representing entire commands!

File: pages/common/tee.md [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
 - Print standard input to the terminal, and also pipe it into another program for further processing:
 
-`echo "example" | tee /dev/tty | xargs printf "\033[1;34m%s\033[m\n"`
+`echo "example" | tee {{/dev/tty}} | {{xargs printf "[%s]"}}`
```
Comment (mfrw) [inline pages/common/tee.md]: I think `echo "example" | tee | xargs printf "[%s]"` will also do the same.
What benefit do we have from including `/dev/tty`

Comment (ticky) [reply pages/common/tee.md]: That’s not correct:

image

As the description says, this both prints to your terminal *and* passes it to the next command. Tee with no args does nothing.

Comment (mfrw) [reply pages/common/tee.md]: I don't know what was I thinking :(
You are correct, I apologise.

**Pull request merged**